### ISSUE TITLE
create PRESS page

### ISSUE THREAD

**Issue by LoChef23** [LoChef23/IMEMOI#9]
[PRESS.zip](https://github.com/LoChef23/IMEMOI/files/5179376/PRESS.zip)

Page should have this structure:
- header
- main content:
   - grid containing all the press news: on each row of the grid is possible to put 4 press news for large screens and less (maybe 3 ofr medium and 2 for small) press news for smaller screens
- footer

Actions: clicking on each press news, the article should be 'focused' allowing the user to read it

Basically it's necessary to copy the behaviour of: https://www.imemoi.com/press/

**Comment by KseniiaPelykh** [comment]
:+1:

**Comment by KseniiaPelykh** [comment]
will work on it today
